BibTeX.js

```diff
@@ -262,7 +263,14 @@ function setKeywordDelimRe( val, flags ) {
 function processField(item, field, value, rawValue) {
 	if(Zotero.Utilities.trim(value) == '') return null;
 	if(fieldMap[field]) {
-		item[fieldMap[field]] = value;
+		//map DOIs + Label to Extra for unsupported item types
```

Comment:
but they're available in the translator object before saving, so something like this in a web translator:
https://github.com/zotero/translators/blob/master/Springer%20Link.js#L158
would cause DOI: 10.1234/123456 to appear twice in the Extra field.